### DISCUSSION TITLE
How to fill radolan_geometry table?

### DISCUSSION THREAD

**Discussion by ghost** [technologiestiftung/giessdenkiez-de#400] [Q&A]
Hi folks,

I want to fill the radolan_geometry table with the data from my hometown but I don't know how.

It feels like the grid.py file got something to do with it, but that one is depending on a grid.asc file which is nowhere to be seen.

And I can't find something about that issue in the wiki.

Thank you very much!

**Comment by JensWinter** [comment]
Hi there.
I also ran into that question. Can anyone help out here?

Thank you!
